.gitlab-ci.yml

```diff
@@ -2,11 +2,14 @@ stages:
   - test
   - build
   - publish
+  - publish-onchain
```

Comment:
why is this an additional stage?

Reply:
should depend on publishing to AWS.
This is not the final version of CI though.